### DISCUSSION TITLE
Endless Sky Artwork Master List (Under Construction)

### DISCUSSION THREAD

**Discussion by Saugia** [endless-sky/endless-sky#7678] [General]
## ES Artwork Master List

***IN PROGRESS***: This list is currently under construction.

***SPOILER WARNING***: This post contains spoilers for Endless Sky content.

This is a comprehensive list of all Endless Sky artwork that has either been recently, in the process of, or is still in need of being remade/updated to the quality of other current and future assets, as well as any requests thereof. It should be noted that artwork for Endless Sky may be subjective to change, but that doesn't mean we are going to constantly have periods of re-doing the artwork. Therefore, please use the guides listed below when referencing artwork.

## Key Guide to ES Artwork
**Satisfactory** : The artwork has been recently or extensively updated/remodeled and is not in need of a remodel any time soon.
**Sufficient**: The artwork is not in need of an extensive update/remodel. A given reason is provided as to why.
**Adequate**: The artwork is in an acceptable position and is not conflicting with other artwork's quality. However, if a presentable alternative with the help of an artwork reviewer, dev, or large portion of the community is shown, it may be considered for an update/remodel.
**Outdated**: The artwork is not up to par with other artwork in Endless Sky and is up for contention of an update/rework.
**Pending**: The artwork is not currently updated to the game yet, but has a remodel either finished or planned by a specific individual.
**Hot Potato**: The artwork is up for contention of an update/rework by request of an artwork reviewer, dev, or large portion of the community, but has a highly contested subject matter, following a given reason.
(NEW): This graphic is recently new to the Endless Sky artwork roster and not in need of an update any time soon (as it has likely been reviewed to having a desired quality among other content).

**Satisfactory** and **Sufficient** can be interpreted the same way- both mean the artwork is fine where it is and does not need an update.

##

***IMPORTANT***: Any artwork done by a specific creator should be held in respect to said creator, and any attempt to rework said artwork should be discussed with them ahead of time, for example, faction leads, artists that have reworked a large group of works, etc.. In the future, the creator may already have a plan to, or would be happily willing to, update a piece when desired.

***NOTE***: This list is currently limited to Ships and pending Outfits/Weapons. In the future, other imagery may be added, such as planets and landscapes.
***NOTE***: For copyright inquiry, please see the copyright file in your local Endless Sky directory. 
***NOTE***: Pers ships and other unique assets are up to their respective owners and are not included in this list.

## Humanity
<details>
  <summary>Ships</summary>

Aerie - *Satisfactory*
Argosy - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Arrow - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Auxiliary - *Satisfactory*
Bactrian - *Satisfactory*
Barb - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Bastion - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Behemoth - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Berserker - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Blackbird - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Bounder - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Boxwing - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Bulk Freighter - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Carrier - *Satisfactory*
Class C Freighter - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Clipper - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Combat Drone - *Satisfactory*
Container Transport - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Corvette - *Satisfactory*
Cruiser - *Satisfactory*
Dagger - *Satisfactory*
Dreadnought - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Dropship - *Adequate*: There has been discussion of a possible remodel of this graphic by @GefullteTaubenbrust2.
Falcon - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Finch - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Firebird - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Flivver - *Satisfactory*
Freighter - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Frigate - *Satisfactory*
Fury - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Gunboat - *Satisfactory*
Hauler - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Hauler II - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Hauler III - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Hawk - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Headhunter - *Sufficient*: This graphic has historically been an icon for the game as it's an example of a tutorial ship and is inspiring to new creators.
Heavy Shuttle - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Kestrel - *Satisfactory*
Lance - *Satisfactory*
Leviathan - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Manta - *Satisfactory*
Marauder Arrow - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Marauder Bounder - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Marauder Falcon - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Marauder Firebird - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Marauder Leviathan - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Marauder Manta - *Satisfactory*
Marauder Quicksilver - *Satisfactory*
Marauder Raven - *Satisfactory*
Marauder Splinter - *Satisfactory*
Modified Argosy - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Mule - *Satisfactory*
Nest - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Osprey - *Satisfactory*
Protector - *Adequate*: This graphic is suitable for the game, but occasionally discussed for a remodel. @beccabunny is the one who remodeled it last.
Quicksilver - *Satisfactory*
Rainmaker - *Satisfactory*
Raven - *Satisfactory*
Roost - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Scout - *Satisfactory*
Shuttle - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Skein - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Sparrow - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Splinter - *Satisfactory*
Star Barge - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Star Queen - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Surveillance Drone - *Satisfactory*
Vanguard - *Adequate*: This graphic is suitable for the game, but occasionally discussed for a remodel. @beccabunny is the one who remodeled it last.
Wasp - *Sufficient*: This graphic's quality fits in with the quality of other designs.
Science Drone - *Sufficient*: This graphic's quality fits in with the quality of other designs.
</details>

<details>
  <summary>Outfits</summary>
</details>

## Hai
<details>
  <summary>Ships</summary>

Aphid - *Adequate*: This graphic is suitable for the game, but the Hai have not had a model update in a while.
Centipede - *Adequate*: This graphic is suitable for the game, but the Hai have not had a model update in a while.
Cicada - *Adequate*: This graphic is suitable for the game, but the Hai have not had a model update in a while.
Flea - *Adequate*: This graphic is suitable for the game, but the Hai have not had a model update in a while.
Geocoris - *Adequate*: This graphic is suitable for the game, but the Hai have not had a model update in a while.
Geodesic Geocoris - *Adequate*: This graphic is suitable for the game, but the Hai have not had a model update in a while.
Grasshopper - *Adequate*: This graphic is suitable for the game, but the Hai have not had a model update in a while.
Ladybug - *Satisfactory* (NEW)
Lightning Bug - *Adequate*: This graphic is suitable for the game, but the Hai have not had a model update in a while.
Pond Strider - *Adequate*: This graphic is suitable for the game, but the Hai have not had a model update in a while.
Scarab - *Adequate*: This graphic is suitable for the game, but the Hai have not had a model update in a while.
Sea Scorpion - *Adequate*: This graphic is suitable for the game, but the Hai have not had a model update in a while.
Shield Beetle - *Adequate*: This graphic is suitable for the game, but the Hai have not had a model update in a while.
Solifuge - *Adequate*: This graphic is suitable for the game, but the Hai have not had a model update in a while.
Violin Spider - *Adequate*: This graphic is suitable for the game, but the Hai have not had a model update in a while.
Water Bug - *Adequate*: This graphic is suitable for the game, but the Hai have not had a model update in a while.
</details>

<details>
  <summary>Outfits</summary>
</details>

## Remnant
<details>
  <summary>Ships</summary>

Albatross - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Gull - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Heron - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Ibis - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Merganser - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Modified Dromedary - *Satisfactory* (NEW)
Pelican - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Penguin - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Peregrine - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Petrel  - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Puffin - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Smew - *Satisfactory* (NEW)
Starling - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Tern - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
</details>

<details>
  <summary>Outfits</summary>
</details>

## Wanderers
<details>
  <summary>Ships</summary>

Autumn Leaf - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to..
Cool Breeze - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Deep River - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Deep River Transport - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Derecho - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Earth Shaper - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Flycatcher - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Hurricane - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Riptide - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Strong Wind - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Summer Leaf - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Tempest - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Winter Gale - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
</details>

<details>
  <summary>Outfits</summary>
</details>

## Coalition
<details>
  <summary>Ships</summary>

Arach Courier - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Arach Freighter - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Arach Hulk - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Arach Spindle - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Arach Transport - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Kimek Briar - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Kimek Spire - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Kimek Thistle - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Kimek Thorn - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Saryd Runabout - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Saryd Sojourner - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Saryd Traveler - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Saryd Visitor - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.

Heliarch Breacher - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Heliarch Hunter - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Heliarch Interdictor - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Heliarch Judicator - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Heliarch Neutralizer - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Heliarch Punisher - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Heliarch Pursuer - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Heliarch Rover - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Heliarch Stalker - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
</details>

<details>
  <summary>Outfits</summary>
</details>

## Korath
<details>
  <summary>Ships</summary>

Arch-Carrack - *Satisfactory* (NEW)
Charm-Shallop - *Satisfactory* (NEW)
Echo-Galleon - *Satisfactory* (NEW)

nra'ret - *Satisfactory* (NEW)
olofez - *Satisfactory*
Ikatila'ej - *Satisfactory* (NEW)
Kas'lor Ik 582 - *Satisfactory*
Kasichara A'awoj - *Satisfactory* (NEW)
Korsmanath A'awoj - *Satisfactory* (NEW)
Lor'kas Ik 577 - *Satisfactory*
Lor'nag Ik 590 - *Satisfactory* (NEW)
Palavret - *Satisfactory*
Ra'at Ik 621 - *Satisfactory* (NEW)
Ra'gru Ik 618 - *Satisfactory*
Rai'alorej - *Satisfactory* (NEW)
Rano'erek - *Satisfactory*
Tubfalet - *Satisfactory* (NEW)

Far Lek 14 - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Far Osk 27 - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Kar Ik Vot 349 - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Met Par Tek 53 - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Tek Far 109 - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Tek Far 71 - Lek - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Tek Far 78 - Osk - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.

Model 128 - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Model 16 - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Model 256 - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Model 32 - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Model 512 - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Model 64 - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Model 8 - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
</details>

<details>
  <summary>Outfits</summary>
</details>

## Ka'het
<details>
  <summary>Ships</summary>

Faes'mar - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Fetri'sei - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Maeri'het - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Selii'mar - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Telis'het - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Vareti'het - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
</details>

<details>
  <summary>Outfits</summary>
</details>

## Quarg
<details>
  <summary>Ships</summary>

Quarg Skylark - *Hot Potato*: The Quarg are next in line for both a visual and conceptual update in-game. However, remodels of Quarg ships are extremely competitive as there are several desired designs for them.
Quarg Wardragon - *Hot Potato*: The Quarg are next in line for both a visual and conceptual update in-game. However, remodels of Quarg ships are extremely competitive as there are several desired designs for them.
</details>

<details>
  <summary>Outfits</summary>
</details>

## Pug
<details>
  <summary>Ships</summary>

Pug Arfecta - *Satisfactory*
Pug Enfolta - *Satisfactory*
Pug Maboro - *Satisfactory*
Pug Zibruka - *Satisfactory*
</details>

<details>
  <summary>Outfits</summary>
</details>

## Drak
<details>
  <summary>Ships</summary>
Archon - *Adequate*: This graphic is suitable for the game, but has not received a model update before. Some believe the graphic to be fine as-is, while others believe it could be enhanced.
</details>

<details>
  <summary>Outfits</summary>
</details>

## Sheragi
<details>
  <summary>Ships</summary>

Black Diamond - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
Emerald Sword - *Sufficient*: This graphic's design is in a preferable state for the faction it belongs to.
</details>

<details>
  <summary>Outfits</summary>
</details>

## Indigenous
<details>
  <summary>Ships</summary>

Ember Waste Node - *Adequate*: This graphic is suitable for the game, but has not received a model update before. Some believe the graphic to be fine as-is, while others believe it could be enhanced.
Embershade - *Sufficient*: This graphic's design is in a preferable state for the area it belongs to.
Pincer Beast - *Satisfactory* (NEW)
Subsidurial - *Adequate*: This graphic is suitable for the game, but has not received a model update before. Some believe the graphic to be fine as-is, while others believe it could be enhanced.
Void Sprite - *Sufficient*: This graphic's design is in a preferable state for the area it belongs to.
</details>

<details>
  <summary>Outfits</summary>
</details>

**Comment by McloughlinGuy** [comment]
I think the wanderer ships are more along the lines of adequate than sufficient, they haven't been updated in quite some time and could probably use a remodel but are still adequate.

**Reply by GefullteTaubenbrust2** [reply]
Agreed, the models aren‘t up to the same standards as say the human ships.

**Reply by Saugia** [reply]
Personally, I love the current look of Wanderer ships. They still retain some of MZ's old alien style while not being as out of date looking as old human designs, and remind me even of EV Override ships. I think it would be horrible to up-detail them like we've done everything else and make everything that much more bland detail-level wise. I had a lot of fun modelling the Squall and such and I feel those look really good and don't need any updates.

**Reply by GefullteTaubenbrust2** [reply]
> Personally, I love the current look of Wanderer ships. They still retain some of MZ's old alien style while not being as out of date looking as old human designs, and remind me even of EV Override ships. I think it would be horrible to up-detail them like we've done everything else and make everything that much more bland detail-level wise. I had a lot of fun modelling the Squall and such and I feel those look really good and don't need any updates.

No one said they were bad, but they are pretty rough around the edges. They very much still have the lo fi MZ chunkiness to them which is fine in itself but a bit out of line with other ships.

**Reply by Hecter94** [reply]
Some of the Wanderer ships could definitely use an update, yes.
